src/components/plotly/pie/index.js

```diff
@@ -21,22 +16,23 @@ const Pie = ({
   hole,
   formatData,
   ...props
-}) => (
-  <CustomPlot
-    type='pie'
-    data={useTransformedData({
-      type: 'pie',
-      data,
-      extra: {
-        textinfo: textinfo ?? getTextInfo({ showPercentage, showLabelName }),
-        hole: hole ?? (donut ? 0.4 : 0),
-      },
-      formatData,
-      ...props,
-    })}
-    {...props}
-  />
-)
+}) => {
```

Comment:
@Hamaad102 @woozyking why was this change necessary? We usually use this short form of returning.

Reply:
Sorry, this change wasn't necessary. I did this in the midst of figuring out what the values being sent to each component were.

Reply:
No worries, I can fix it in my next PR, as I have some local changes already.